alkira/resource_alkira_connector_azure_er.go

```diff
@@ -0,0 +1,294 @@
+package alkira
+
+import (
+	"fmt"
+
+	"github.com/alkiranet/alkira-client-go/alkira"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceAlkiraConnectorAzureEr() *schema.Resource {
+	return &schema.Resource{
+		Description: "Manage Azure ExpressRouter Connector.",
+
+		Create: resourceConnectorAzureErCreate,
+		Read:   resourceConnectorAzureErRead,
+		Update: resourceConnectorAzureErUpdate,
+		Delete: resourceConnectorAzureErDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Description: "The name of the connector.",
+				Type:        schema.TypeString,
+				Required:    true,
+			},
+			"credential_id": {
+				Type:     schema.TypeString,
```

Comment:
Description is gone? I think this one will share the `credential_azure_vnet`?

Reply:
My mistake. This field is not needed at all. The credential_id is in the instances block which has the right description.